es-cluster-deployment.yml

```diff
@@ -101,6 +101,7 @@ spec:
   selector:
     app: elasticsearch
   type: ClusterIP
+  clusterIP: None
```

Comment:
heads up: I have added it here (at the moment it's not in 5.x)

https://github.com/RHsyseng/docker-rhel-elasticsearch/issues/9#issuecomment-381153548